edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.E2E.Test/test-config-x64.json

```diff
@@ -34,7 +34,7 @@
             "$type": "RunCommandValidator",
             "command": "curl",
             "args": "-X GET -s -o /dev/null -w \"%{http_code}\" --unix-socket /var/run/docker.sock http://docker/containers/bash/json",
-            "exitCode" : 23,
+            "exitCode" : 0,
```

Comment:
Is 0 the right value?
@arsing

Reply:
It should succeed, so yes. `23` apparently only happens if it can't write the output, like if stdout is closed, so I wonder how it was 23 in the first place.

Reply:
I was getting that error code when I ran similar test cases on my windows machine as it was failing to write to stdout for some reason although the output data matched what I expected. Not entirely sure why.

Reply:
Strange. `RunCommandValidator` definitely [lets the process write all the output it wants](https://github.com/Azure/iotedge/blob/f9fc465a4d7c5c9c6faa7ecf231444516236e889/edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.E2E.Test/RunCommandValidator.cs#L34-L39) before evaluating the match. Doesn't matter anyway.